electron/src/renderer/preload-app.ts

```diff
@@ -99,7 +99,7 @@ const setupIpcInterface = (): void => {
       }
 
       console.info(`Processing deletion of "${accountID}"`);
-      const viewInstanceId = accountWebview.getWebContents?.().id;
+      const viewInstanceId = accountWebview.getWebContentsId();
```

Comment:
I think that breaks when deleting an account while being on the error page (no webview is rendered)